include/crpropa/Grid.h

```diff
@@ -60,18 +69,8 @@ class GridProperties: public Referenced {
 	GridProperties(Vector3d origin, size_t N, double spacing) :
 		origin(origin), Nx(N), Ny(N), Nz(N), spacing(Vector3d(spacing)), reflective(false) {
 	}
-	
-	/** Constructor for non-cubic grid
-	 @param	origin	Position of the lower left front corner of the volume
-	 @param	Nx		Number of grid points in x-direction
-	 @param	Ny		Number of grid points in y-direction
-	 @param	Nz		Number of grid points in z-direction
-	 @param spacing	Spacing between grid points
-	 */
-	GridProperties(Vector3d origin, size_t Nx, size_t Ny, size_t Nz, double spacing) :
```

Comment:
This constructor shows up as missing in the diff, and I just checked the CRPropa master and it has this constructor as well. Is there a reason it was deleted?